lib/matplotlib/axes/_base.py

```diff
@@ -444,7 +442,7 @@ def __init__(self, fig, rect,
         self.set_label(label)
         self.set_figure(fig)
         self.set_box_aspect(box_aspect)
-        self.set_axes_locator(kwargs.get("axes_locator", None))
+        self._axes_locator = None
```

Comment:
maybe:
```suggestion
        self._axes_locator = None  # set later via self.update(kwargs)
```

Reply:
Not necessarily, it can be left to None.

Reply:
Optionally set...

The idea marking it clear that this parameter may be initialized via kwargs update. IMHO this is slightly unusual and not transparent to the reader of the code.

Reply:
sure, done